app/src/main/java/com/andryoga/safebox/ui/view/login/LoginFragment.kt

```diff
@@ -44,6 +44,21 @@ class LoginFragment : Fragment() {
 
         setupObservers()
 
+        binding.ShowHintText.setOnClickListener {
+
+            if (binding.ShowHintText.text == getString(R.string.show_hint)) {
+
+                binding.ShowHintText.text = getString(R.string.hide_hint)
+                binding.displayHintText.visibility = View.VISIBLE
```

Comment:
instead of manually hiding and showing view you can use this util method. 
https://github.com/Ni3verma/Safe-Box/blob/f1af06060ae468d31b1ca542044da137f94e440d/app/src/main/java/com/andryoga/safebox/ui/common/Utils.kt#L7

its cleaner.